lib/ui/welcome/payable/payable_viewmodel.dart

```diff
@@ -35,11 +35,30 @@ class PayableViewModel extends BaseViewModel {
     return _payable;
   }
 
+  // initializing state to work with button on keypad
+  String _amount = '';
+  String get amount {
+    return _amount;
+  }
+
+  bool _showButton = false;
+  bool get showButton {
+    return _showButton;
+  }
+
   int _total;
   int get total {
```

Comment:
Let's make it double as the app will be used globally other currencies might need cents.